backend.go

```diff
@@ -21,6 +21,22 @@ type Backend interface {
 	AnonymousLogin(state *ConnectionState) (Session, error)
 }
 
+type Feature int32
+
+const (
+	FeatureSMTPUTF8 Feature = 2 << iota
```

Comment:
Can we keep the references to the RFCs that were in the `Server` fields?

Reply:
Done.